### PR TITLE
Remove mixer/telemetry-v1 use of *_app and *_version metric attributes

### LINKED ISSUE #3340: Remove mixer/telemetry-v1 use of _app and _version metric attributes
Epic #3328

### PR DESCRIPTION
**Pull request by jshaughn**
We can now rely on *_canonical_service and *_canonical_revision attributes.

closes #3340

### CODE REVIEW

File: handlers/config.go [hunk 1]
```diff
@@ -56,7 +55,6 @@ type PublicConfig struct {
 	IstioNamespace           string                          `json:"istioNamespace,omitempty"`
 	IstioComponentNamespaces config.IstioComponentNamespaces `json:"istioComponentNamespaces,omitempty"`
 	IstioLabels              config.IstioLabels              `json:"istioLabels,omitempty"`
-	IstioTelemetryV2         bool                            `json:"istioTelemetryV2"`
```
Comment (jotak) [inline handlers/config.go:59]: So I guess there must be a front-end PR ?

**Review by jotak — COMMENTED** [review]
Good cleaning, almost gtm except I would expect a frontend PR related to public config, right?

File: handlers/config.go (hunk 1 above)
Comment (lucasponce) [reply handlers/config.go:59]: Mmm note that it's also modified in the other PRs used for Mixer removal, I guess here there will be a minor conflict between both efforts.

**Comment by lucasponce** [comment]
> Good cleaning, almost gtm except I would expect a frontend PR related to public config, right?

Yes, it's being addressed on https://github.com/kiali/kiali/pull/3344 et al, so I guess we will solve the conflict when all PRs are ready to consolidate.

**Comment by lucasponce** [comment]
I probably created a minor conflict on this PR as I guess I touched same file, but I guess it's an easy rebase on this case.

File: handlers/config.go (hunk 1 above)
Comment (jshaughn) [reply handlers/config.go:59]: Right, I will rebase and I think we'll be OK.

**Comment by jshaughn** [comment]
@lucasponce @jotak This PR is rebased and no longer modifies config.go, which was updated in Lucas' PRs for both server and ui.  So I think it's good to go.

@mattmahoneyrh This PR needs only standard regression testing, mainly around graph and charts.

**Review by jotak — APPROVED** [review]
LGTM

**Comment by lucasponce** [comment]
Code LGTM

**Review by mattmahoneyrh — APPROVED** [review]

**Pull request merged**